### DISCUSSION TITLE
Using min. 2 different input type

### DISCUSSION THREAD

**Discussion by RanggaGemilang** [teamalgoritma/community#255] [Q&A] [answered]
Halo, 

Saya ingin bertanya apakah **selectInput()** dan **checkboxGroupInput()** akan dikategorikan 2 tipe yang berbeda dalam penilaian?

**Comment by amaliasf** [comment, marked as answer]
Dear Pak @RanggaGemilang 

Benar sekali Pak, `selectInput()` dan `checkboxGroupInput()` akan dikategorikan menjadi 2 tipe input yang berbeda dalam penilaian. Semoga membantu Pak :)
